DATA/production/configurations/asyncReco/setenv_extra.sh

```diff
@@ -427,6 +427,12 @@ fi
 if [[ $ALIEN_JDL_DISABLECASCADES == 1 ]]; then
   export ARGS_EXTRA_PROCESS_o2_secondary_vertexing_workflow+=" --disable-cascade-finder  "
 fi
+# allow usage of TPC-only in svertexer (default: do not)
+if [[ $ALIEN_JDL_USETPCONLY == 1 ]]; then
```

Comment:
@ddobrigk , can you make this env var more explicit? E.g. "ALIEN_JDL_USETPCONLYFORV0S" or so?